libvips/iofuncs/header.c

```diff
@@ -1451,7 +1451,7 @@ add_fields( VipsImage *image, const char *field, GValue *value, void *a )
 }
 
 /**
- * vips_image_get_fields: (method)
+ * vips_image_get_fields:
```

Comment:
Isn't this a method? I'm not sure I understand.

Reply:
having a `(method)` forces it to be a method, here as it is already prefixed by `vips_image_` and has `VipsImage *` as first argument there is no ambiguity.
Forcing it to be a method actually makes the method to be `Vips.Image.image_get_fields` when not forcing it makes the method name be `Vips.Image.get_fields`.

Reply:
Ah gotcha. Thanks for the explanation.